DEVDOCS-4571: [new] B2B Edition for Stencil + Scripts + MSF Guide, add references to B2B Buyer Portal + MSF-compatibility (#1916)
bigcommerce/dev-docs: docs/stencil-docs/installing-stencil-cli/installing-stencil.mdx

</Callout>

> Stencil CLI gives developers the power to locally edit and preview themes with no impact to a merchant's live storefront, and its built-in [Browsersync](https://github.com/bigcommerce/browser-sync) capabilities make simultaneous testing across desktop, mobile, and tablet devices a breeze. Once work is complete, developers can push themes to BigCommerce storefronts and make them live using Stencil CLI's simple yet powerful commands.

Reviewer: ```suggestion
Stencil CLI allows developers to locally edit and preview themes without impacting a merchant's live storefront, and its built-in [Browsersync](https://github.com/bigcommerce/browser-sync) capabilities make simultaneous testing across desktop, mobile, and tablet devices a breeze. Once work is complete, developers can push themes to BigCommerce storefronts and make them live using Stencil CLI's simple yet powerful commands.

```

Author: closer, thanks!

> This article contains the detailed instructions needed to install and configure Stencil CLI - the first step to developing themes on the BigCommerce platform.

Reviewer: ```suggestion
This article contains detailed instructions to install and configure Stencil CLI - the first step to developing themes on the BigCommerce platform.

```

Author: i like this and want to take it further.

```suggestion
This article contains detailed instructions on installing and configuring Stencil CLI, the first step towards developing Stencil themes for BigCommerce storefronts.

```
